### DISCUSSION TITLE
Checkout failing to cleanup temp file

### DISCUSSION THREAD

**Discussion by StephenHodgson** [community/community#57973] [Actions] [answered]
### Select Topic Area

Bug

### Body

Getting this error when checking out my private repo on a self hosted runner:
```
E:\runner\w\_actions\actions\checkout\v3\dist\index.js:6049
            throw new Error(`File was unable to be removed ${err}`);
                  ^

Error: File was unable to be removed Error: EBUSY: resource busy or locked, rmdir 'E:\runner\w\_temp\1f0f130c-9a04-494f-ace1-7162a18ec8'
    at Object.<anonymous> (E:\runner\w\_actions\actions\checkout\v3\dist\index.js:6049:19)
    at Generator.throw (<anonymous>)
    at rejected (E:\runner\w\_actions\actions\checkout\v3\dist\index.js:43:65)
```

I have not changed any configuration, and I already tried to remove then add runner back to the organization

**Comment by StephenHodgson** [comment, marked as answer]
So after doing some digging I figured out that I had recently re-installed git for windows, and enabled the new file watcher feature. This actually was causing my _temp folder to become locked and prevent the checkout action from completing successfully.

**Reply by LaithDaman** [reply]
I am having exactly the same issue. When I start runner from my local machine, it works. On our AWS and another machine; I get this exact error. I am interested in seeing how did you fix it? I am noob at this so I appreciate the extra steps.

**Reply by StephenHodgson** [reply]
I just re-installed git without file watcher feature

**Comment by vladimir-voinea** [comment]
Running this in PowerShell as Administrator will fix it:

`git config --system core.usebuiltinfsmonitor false`

**Reply by StephenHodgson** [reply]
This is also a perfectly acceptable answer and is better than my solution which was completely doing a re-install of git on the machine, lol.

**Comment by ZVV1971** [comment]
I've just found out that when I removed the folder from the gitconfig.safe section it worked fine again.
